### ISSUE TITLE
Torrents list in latest uTorrent not read correctly by NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4009]
**Reported by k_kolev1985 on 2014-03-21 13:33**
Under Windows 8.1 Pro N 64-bit, NVDA does not read the list of torrents in latest uTorrent.

Steps to reproduce it:
1. Install the latest stable version of uTorrent: 3.4.0.30660.
2. Load a list of torrents.
3. Move the keyboard/system focus to the list of torrents with the TAB key, if it is not already there.
4. Move the keyboard/system focus in the list of torrents with the up/down arrow keys.

Actual results:
NVDA does not read the name of the currently selected torrent in the list. It does read its number and I think all other information, but not the name.

Expected results:
NVDA should read the name of the torrent for the currently selected item.

My system specifications: 
- OS: Windows 8.1 Pro N, 64-bit, in bulgarian. 
- NVDA version next-10440,0517f65; 2014.1.
- uTorrent version: 3.4.0.30660 (in its titlebar it says that it is a 32-bit version, but I can't find a 64-bit one on its site).

**Comment by bhavyashah** [comment]
@k_kolev1985  Do you still experience the reported issue in the presently latest version of uTorrent with NVDA 2017.3?

**Comment by k-kolev1985** [comment]
Yes, the bug is still present, as I mentioned in another ticket here. The testing was done with uTorrent 3.5 build 43916 32-bit and the one of the latest "next" snapshots of NVDA.

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, the issue is still reproducible. And no - I haven't contacted the uTorrent developers about it and I'll probably will not do it, because I doubt it they will take it in consideration at all.

I think a fix for broken reading of list views in 64-bit programs (particularly "TortoiseSVN") has managed to fix this. But that fix was lost while changing from "next" to "alpha" snapshots (which happened some time in July) and it is still not committed again.

**Comment by Adriani90** [comment]
I
ve tested with NVDA 2022.4 and utorrent 3.6.0, the list can be accessed with NVDA just fine. I am closing this issue as works for me, please feel free to comment in case you still have the issue with NvDA 2022.4 and the last version of Utorrent. Note that they also have a web client which is very accessible.
